packages/opticss/src/query.ts

```diff
@@ -102,32 +102,32 @@ export class QuerySelectorReferences implements SelectorQuery {
  * The returned selectors may not actually match depending on the selector
  * context and various combinators.
  */
-export class QueryKeySelector implements SelectorQuery {
```

Comment:
This entire class is un-used....should I remove it? Seems it may be important later.

This is one of a few reasons why I want to break selector parser and query into their own packages. They're better candidates for unit tests than integration tests because they provide single-domain utility APIs.

Reply:
QueryKeySelector is used in css-blocks.